src/main/java/school/lemon/changerequest/java/Employee.java

```diff
@@ -0,0 +1,50 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 15.12.16.
+ */
+public class Employee {
+    private String name;
+    private double salary;
+    private int workedHours;
+    public static final int hoursInMonth = 160;
+
+    public Employee() {
+    }
+
+    public Employee(String name) {
+        this.name = name;
+    }
+
+    public void setWorkedHours(int workedHours) {
+        this.workedHours = workedHours;
+    }
+
+    public void setSalary(double salary) {
+        this.salary = salary;
+    }
+
+    public double getSalary() {
+        return salary;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+
+    public double getRatioOfWorkedHours() {
+        double ratioHours = (double) workedHours / hoursInMonth;
+        return ratioHours;
+    }
+
+    public double calculateSalary() {
```

Comment:
In my opinion, this method should be abstract, as only Programmer use it directly.